src/components/drivers/pdf-viewer.jsx

```diff
@@ -103,17 +107,20 @@ export default class PDFDriver extends React.Component {
   }
 
   componentDidMount() {
-    const { filePath } = this.props
-    const containerWidth = this.container.offsetWidth
-
-    const loadingTask = PDFJS.getDocument(filePath)
-    loadingTask.onProgress = (progressData) => {
-      this.progressCallback(progressData)
+    // Only utilized PDFJS.getDocument() if isEvalSupported == false.
```

Comment:
utilized here should be utilize, right?

Reply:
Yes, yes it should. Thanks for spotting that